app/frontend/src/uploadDocuments.js

```diff
@@ -1,8 +1,22 @@
-// This triggers a file input when the button is clicked
-
 document.addEventListener('DOMContentLoaded', function() {
-  document.getElementById('file-upload').addEventListener("click", function(e) {
+  inputFileUpload = document.getElementById('documents-form-documents-field')
+    ? document.getElementById('documents-form-documents-field')
+    : document.getElementById('documents-form-documents-field-error')
+
+  selectFileButton = document.getElementsByClassName('govuk-button--secondary')[0];
+  uploadFileButton = document.getElementsByClassName('govuk-button--secondary')[1];
+
+  selectFileButton.addEventListener('click', function(e) {
```

Comment:
Do we have on-page JS error monitoring with one of our alerting services?  If so, then this is all fine, we'll get alerts if the expected elements aren't in the dom.  If not, then we probably need to set something up (sentry.io is good) and plug it in before we release this.

Reply:
We do have JS errors monitoring:

https://rollbar.com/dfe/teacher-vacancies

Reply:
:+1: